src/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py

```diff
@@ -22,7 +22,7 @@
 from toil_scripts.lib.urls import download_url_job, s3am_upload
 from toil_scripts.tools.QC import run_fastqc
 from toil_scripts.tools.aligners import run_star
-from toil_scripts.tools.preprocessing import cutadapt
+from toil_scripts.tools.preprocessing import run_cutadapt
```

Comment:
Ah, no big deal then. It doesn't offend me, I just wanted to make sure it wasn't accidentally in the wrong place.